mavencentral-publish.gradle

```diff
@@ -0,0 +1,102 @@
+apply plugin: 'maven-publish'
+apply plugin: 'signing'
+apply from: buildscript.sourceFile.getParent() + "/" + "mavencentral-properties.gradle"
+
+// The publications doesn't know about our AAR dependencies, so we have to manually add them to the pom
```

Comment:
Is this still the case? Thought it was fixed somewhere around AGP 3.1/2. Anyway, if this works, just keep it. But maybe put a comment about reverifying instead of maintaining it.

Reply:
I think it is still the case as as I accidentally tested it during the migration.

Reply:
Weird. Maybe our publications doesn't fit exactly to the expectations of AGP 🤷